### ISSUE TITLE
Segfault on Debian SID

### ISSUE THREAD

**Issue by wargreen** [OSSIA/OSSIA#11]
On a build from the GIT, on debian SID x64, I got lot of segfault, when try to  use i-score and add a child parameter.

strace
[....]
read(5, "\1\0\0\0\0\0\0\0", 16)         = 8
write(5, "\1\0\0\0\0\0\0\0", 8)         = 8
poll([{fd=3, events=POLLIN|POLLOUT}], 1, 4294967295) = 1 ([{fd=3, revents=POLLOUT}])
writev(3, [{"+\0\1\0", 4}], 1)          = 4
futex(0x7ffcf3887124, FUTEX_WAIT_PRIVATE, 1, NULL) = -1 EAGAIN (Resource temporarily unavailable)
futex(0x229ed38, FUTEX_WAKE_PRIVATE, 1) = 0
poll([{fd=3, events=POLLIN|POLLOUT}], 1, 4294967295) = 1 ([{fd=3, revents=POLLOUT}])
writev(3, [{"\202\3\n\0007\0@\3<\0@\3\23\1'\1b\0\5\1P\0\27\0b\0\5\1\30\2\0\0"..., 40}], 1) = 40
poll([{fd=3, events=POLLIN|POLLOUT}], 1, 4294967295) = 1 ([{fd=3, revents=POLLOUT}])
writev(3, [{"\n\0\2\0007\0@\3\31\0\v\0\227\2\0\0\0\0\30\0\22cQ\2\227\2\0\0007\0@\3"..., 52}], 1) = 52
poll([{fd=3, events=POLLIN|POLLOUT}], 1, 4294967295) = 1 ([{fd=3, revents=POLLIN|POLLOUT}])
recvmsg(3, {msg_name(0)=NULL, msg_iov(1)=[{"\n\3\23'7\0@\3\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"..., 4096}], msg_controllen=0, msg_flags=0}, 0) = 64
futex(0x229ed64, FUTEX_WAKE_OP_PRIVATE, 1, 1, 0x229ed60, {FUTEX_OP_SET, 0, FUTEX_OP_CMP_GT, 1}) = 1
writev(3, [{"&\0\2\0\227\2\0\0", 8}], 1) = 8
futex(0x229ed38, FUTEX_WAKE_PRIVATE, 1) = 1
poll([{fd=3, events=POLLIN}], 1, 4294967295) = 1 ([{fd=3, revents=POLLIN}])
recvmsg(3, {msg_name(0)=NULL, msg_iov(1)=[{"\10\0\23'\342`v\17\227\2\0\0007\0@\3\0\0\0\0\321\3\246\2\232\0\r\1\20\0\0\3", 4096}], msg_controllen=0, msg_flags=0}, 0) = 32
futex(0x229ed64, FUTEX_WAKE_OP_PRIVATE, 1, 1, 0x229ed60, {FUTEX_OP_SET, 0, FUTEX_OP_CMP_GT, 1}) = 1
futex(0x229ed38, FUTEX_WAKE_PRIVATE, 1) = 1
poll([{fd=3, events=POLLIN}], 1, 4294967295) = 1 ([{fd=3, revents=POLLIN}])
recvmsg(3, {msg_name(0)=NULL, msg_iov(1)=[{"\1\1\25'\0\0\0\0\227\2\0\0\17\266\343\0\321\3\246\2\321\3\246\2\20\0\0\0\0\0\0\0", 4096}], msg_controllen=0, msg_flags=0}, 0) = 32
futex(0x229ed64, FUTEX_WAKE_OP_PRIVATE, 1, 1, 0x229ed60, {FUTEX_OP_SET, 0, FUTEX_OP_CMP_GT, 1}) = 1
futex(0x229ed38, FUTEX_WAKE_PRIVATE, 1) = 1
write(5, "\1\0\0\0\0\0\0\0", 8)         = 8
--- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=0} ---
+++ killed by SIGSEGV +++
[1]    23324 segmentation fault  strace ./i-score

**Comment by theod** [comment]
thanks for reporting this @wargreen.
however I'm not sure if @jcelerier will be able to take care of this issue soon as he is fully engage into the next version development.
is the application downloaded from http://i-score.org site also crashes ?

**Comment by wargreen** [comment]
I can't install the JamomaCore packiage from your .zip, cause of dependencies problem on SID
Dépaquetage de jamomacore (0.6-dev) sur (0.6-dev) ...
dpkg: des problèmes de dépendances empêchent la configuration de jamomacore :
 jamomacore dépend de libgecode36 | libgecode32 | libgecode30 ; cependant :
  Le paquet libgecode36 n'est pas installé.
  Le paquet libgecode32 n'est pas installé.
  Le paquet libgecode30 n'est pas installé.


apt-get install libgecode36
Package libgecode36 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libgecode36' has no installation candidate

**Comment by jcelerier** [comment]
Yes, it seems that gecode has been bumped to libgecode41 on sid. Sadly this means that the gecode API has changed and this would certainly involve changes in the gecode binding code in Jamoma, code that has been deprecated about 6 months ago. So there won't be new work on this anytime soon.

However, by the end of summer there should be something testable for the next version (which does not use gecode and has less dependencies overall), with many more user friendly features (undo-redo, etc...) if you can afford to wait :)
